### DISCUSSION TITLE
LinkedIn - HTTP 403 for profile image

### DISCUSSION THREAD

**Discussion by AydinChavez** [nextauthjs/next-auth#3496] [Help]
### Question 💬

I migrated next-auth from 4.0.0-beta.2 to 4.0.5 and am using LinkedIn as Provider.
With that upgrade, I also removed the logic for retrieving email address and profile image from linkedin (profile method), since it's done now by next-auth automatically.

Authentication still works fine, but I noticed that the returned profile image is not accessible anymore (although there is a profile image url returned from the next-auth provider logic) - I get a http 403.

My setup is quite simple, am just passing client_id and secret (since the rest is populated from LinkedIn by Next-Auth). 
Is there anything I should consider here or does anybody else have the same issue? 

### How to reproduce ☕️

My session object after the login looks valid (I anonymized the content):

```
session {
  "user": {
    "name": "Aydin xx",
    "email": "aydin.xx@t-online.de",
    "image": "https://media-exp1.licdn.com/dms/image/C4E03AQHhwS8aDgqy0A/profile-displayphoto-shrink_800_800/0/xxxxxx?e=1637193600&v=beta&t=xxxx"
  },
  "expires": "2022-01-20T14:34:18.126Z"
}
```

### Contributing 🙌🏽

No, I am afraid I cannot help regarding this

**Comment by balazsorban44** [comment]
I just tested this locally, and I did get the image, without 403. If I remember correctly, there was some kind of verification you had to go through at LinkedIn for it to show up. @ndom91 maybe you remember? Once figured out, we should add this info to https://next-auth.js.org/providers/linkedin
